spacepy/datamanager.py

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-The datamanager classes and functions are useful for locating the correct
-data file for a particular day and manipulating data and subsets in a generic
-way.
+Tool for locating the correct data file for a particular day and manipulating data and subsets generically.
```

Comment:
Yeah the reason I made this change was specifically to fit them onto one line, although you may be right, in the code, it may still be better to split it between lines.